fellowship/cli.py

```diff
@@ -4,31 +4,47 @@
 
 import argparse
 import json
+import logging
 import sys
 from .contract_generator import ContractGenerator
 from .rest_tester import RestTester
 
 
-def parse_args(args):
+logging.basicConfig(level=logging.INFO)
+
+
+def get_parser():

```

Comment:
Optionally this parser could be transferred to another module. The reason is that then there is no global configuration for logging set.